apps/studio/pages/integrations/vercel/install.tsx

```diff
@@ -175,6 +175,10 @@ const VercelIntegration: NextPageWithLayout = () => {
       : false
   }, [installed, selectedOrg, source, dataLoading])
 
+  const hasVercelManagedOrgSelected = useMemo(() => {
+    return !!selectedOrg && selectedOrg.slug.startsWith('vercel_')
```

Comment:
```suggestion
    return !!selectedOrg && selectedOrg.managed_by === 'vercel-marketplace'
```

Reply:
Oh nice, I didnt see that in the request response as it's set in the code itself.